### DISCUSSION TITLE
how is Coalton pronounced?

### DISCUSSION THREAD

**Discussion by SequentialDesign** [coalton-lang/coalton#843] [Other]
how is Coalton pronounced? is it like Colten? or Co-altin (like co-opt)?

TIA.

**Comment by eliaslfox** [comment]
It's pronounced as Coal-tun. The pronunciation comes from [MLton](http://www.mlton.org).

**Reply by stylewarning** [reply]
(And, incidentally, MLton is pronounced like "mill-tun" or the name Milton.)
